### PR TITLE
Add Submit Button to Review Due Date Changes

### PR DESCRIPTION
**Pull request by chrstinalin**
Fixes: mozilla/addons#14922

### Description

Changes the due date update functionality on the reviewer page to update `onblur` rather than on change.

**Before**
image

**After**
ScreenRecording2024-08-09at8 24 41AM-ezgif com-video-to-gif-converter
Flashes in the gif are tab refreshes. Changes don't persist across reload unless OK is pressed.

### Checklist

- [x] Add `#ISSUENUM` at the top of your PR to an existing open issue in the mozilla/addons repository.
- [x] Successfully verified the change locally.
- [x] The change is covered by automated tests, or otherwise indicated why doing so is unnecessary/impossible.
- [x] Add before and after screenshots (Only for changes that impact the UI).

### CODE REVIEW

**Comment by eviljeff** [comment]
> The button doesn't technically do anything (clicking it just unfocuses the date input and fires the `onblur` event). It's there as a visual aid to distinguish between 'saved' and 'unsaved' states.

I lead towards removing the button because it could be confusing that the date changed before the button was pressed, but it's @wagnerand 's call.  So the options are:
- trigger the XHR on the input blur, with a fake button (the PR now)
- trigger the XHR on the input blur (no button - the reviewer would have to click somewhere else to send the update)
- trigger the XHR on the button being pressed (no event on input blur)

File: src/olympia/reviewers/templates/reviewers/review.html [hunk 1]
```diff
@@ -299,11 +299,14 @@ <h3 id="history">
     <ul class="admin_only">
       <li {% if not version.due_date %}class="hidden"{% endif %}>
         <label for="due_date_update">Change latest version review due date</label>
-        <input type="datetime-local" id="due_date_update"
+        <div style="display: flex; gap: 0.5em">
```
Comment (eviljeff) [inline src/olympia/reviewers/templates/reviewers/review.html:302]: give a class to the div, and add this css to `reviewers.less`

**Comment by wagnerand** [comment]
Let's do

> trigger the XHR on the button being pressed (no event on input blur)

**Review by eviljeff — APPROVED** [review]
r+

**Pull request merged**